flytekit/sdk/sagemaker/types.py

```diff
@@ -0,0 +1,33 @@
+import enum
+
+
+class HyperparameterScalingType(enum.Enum):
```

Comment:
These should all be in the models directory.  Take a look at the `class ResourceType`